src/style.css

```diff
@@ -0,0 +1,18 @@
+body,
+h1,
+p,
+img {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
for this part of code you can make separate file such as `reset.css` and just connect it to your `style.css`

Reply:
done